### ISSUE TITLE
Build failure: anki 24.04

### ISSUE THREAD

**Issue by AMS21** [NixOS/nixpkgs#309200]
### Steps To Reproduce

1. build anki on unstable

### Build log

```
error: builder for '/nix/store/c4sbv8m60478iali4psh8f7dp2hnvz2z-python3.11-mypy-protobuf-3.6.0.drv' failed with exit code 1;
       last 10 log lines:
       > adding 'mypy_protobuf-3.6.0.dist-info/top_level.txt'
       > adding 'mypy_protobuf-3.6.0.dist-info/RECORD'
       > removing build/bdist.linux-x86_64/wheel
       > Successfully built mypy_protobuf-3.6.0-py3-none-any.whl
       > Finished creating a wheel...
       > Finished executing pypaBuildPhase
       > Running phase: pythonRuntimeDepsCheckHook
       > Executing pythonRuntimeDepsCheck
       > Checking runtime dependencies for mypy_protobuf-3.6.0-py3-none-any.whl
       >   - protobuf>=4.25.3 not satisfied by version 4.24.4
       For full logs, run 'nix log /nix/store/c4sbv8m60478iali4psh8f7dp2hnvz2z-python3.11-mypy-protobuf-3.6.0.drv'.
error: 1 dependencies of derivation '/nix/store/sial2j59avq915nmh9nb1rv8p4nhdgb9-python3-3.11.9-env.drv' failed to build
error: 1 dependencies of derivation '/nix/store/dgqnc732xz1rfn5qzds010jslvl63hzk-anki-pyenv-24.04.drv' failed to build
error: 1 dependencies of derivation '/nix/store/16gj0knzqm6y21l9vkc2hynpcyi78vxh-anki-24.04.drv' failed to build
```

[full log](https://gist.github.com/AMS21/bc28ba42aba558fd39002db3107ff648)

### Additional context

None

### Notify maintainers

@euank
@oxij

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.8.8, NixOS, 24.05 (Uakari), 24.05.20240502.63c3a29`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - nixpkgs: `/nix/store/p69bcs7ma6ijj8v9xsrg3nq3nn8ryn95-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by AMS21** [comment]
A workaround is to use the `anki-bin` package instead

**Comment by timokau** [comment]
Should be fixed by #309080. Track it [here](https://nixpk.gs/pr-tracker.html?pr=309080)
